_posts/2021-07-02-migrating-from-legacy-drupal-to-federalist.md

```diff
@@ -0,0 +1,97 @@
+* Robust ATO
+* USWDS Starter kits with built-in integrations with Search.gov and GSA Digital Analytics Program.
+* Helpful support
+
+## When is Federalist right for you?
+
+If your agency is considering migrating away from Drupal, Federalist might be a good choice for you if:
+
+* You are managing an informational site, with frequent updates and changes
+* You are using Github (or are able to), and want more control over versioning of your website
+* You need to support both technical and non-technical content creators / reviewers / editors
+* You need a highly-performant site
+* You have limited capacity to manage infrastructure 
+* You need a simplified ATO process*
+
+## When is Federalist not right for you?
+
+Federalist does have some limitations, so depending on your agency's specific use of Drupal, Federalist may not be a good fit. If you need to do the following, Federalist may not be a viable migration target from Drupal:
+
+* You need to authenticate users, and manage user accounts
+* You need to show different content to different users (user authorization)
+* You need a server (or something that as the characteristics of a server)
+* You have other unique requirements (e.g., software licensing)
+
+## Getting from Drupal to Federalist
+
+Migrating from Drupal to Federalist will be different for each agency using Drupal, based on how your current Drupal website is configured and managed. Generally speaking, the following steps encompass the process of moving from Drupal to the Federalist platform:
+
+* Review / audit existing content
+* Identify appropriate migration tool (lots of options)
+* Modify and update content (as needed)
+* Identify static site generator
+* Choose a starter template or design system
+* Bring content into Federalist site framework
+* Publish to Federalist (push to GitHub)
+
+## Issues and Considerations
+
+Before migrating, there are a number of factors you should take into consideration.
+
+* You may need to conduct a content inventory so you'll know what materials you currently have, and what you need to migrate over
+* Check your list of current enabled modules, to see if any provide functionality critical to your site 
+    - Modules can provide added functionality that you want to make sure you have accounted for.
+    - Some modules (ex: paragraphs) can add extra markup. Make sure you have accounted for this.
+* Plan for taxonomic structure (11ty / netlify / static CMSs almost all have tagging structure)
+* Update any in-content links to new links
+* Migrating users, permissions, etc.
+* Working out a new workflow for publishing
+* Form submits (does your site use them currently)
+
+## Find out more 
+
+The cloud.gov and Federalist team is ready to help you as you evaluate your options for current Drupal 7 and 8 websites. 
+
+In addition, the following resources are also available for agencies:
+
+* [Federalist](https://federalist.18f.gov/)
+* Static site generators
```

Comment:
Change links to open in a new tab or window rather than refreshing the page.

Reply:
Done